lib/internal/test_runner/runner.js

```diff
@@ -96,29 +102,59 @@ function createTestFileList() {
   return ArrayPrototypeSort(ArrayFrom(testFiles));
 }
 
+function isUsingInspector() {
+  return ArrayPrototypeSome(process.execArgv, (arg) => RegExpPrototypeExec(kInspectArgRegex, arg) !== null) ||
+    RegExpPrototypeExec(kInspectArgRegex, process.env.NODE_OPTIONS) !== null;
+}
```

Comment:
this kind of seems like it should just be a boolean property available on `process` or something